surveys/2025-04/questions.md

```diff
@@ -0,0 +1,127 @@
+  - Other?
+
+## Node.js Usage
+
+- Which groups do you identify with? (multiple choice)
+  - Node.js core maintainers: Developers working directly on Node.js, individuals participating in Working Groups
+  - Library & package authors: Users who write libraries and packages to be included on other applications
+  - Application developers: Frontend tools consumers, backend server authors, tools authors
+  - Direct end users: Users who run tools themselves
+  - Application operators: Service and infrastructure providers
+  
+- Are you part of an organization with investments in Node.js? (eg: Enterprises, Gouvernement bodies, Startups, Non-profit, Education, Security)
+  - yes
+  - no
+
+
+- What is your use cases of Node.js ? (multiple choice) (open question - Other should give an input)
+  - Cli tools
+  - Testing
+  - Development of APIs with Microservices
+  - Development of APIs with Serverless
+  - Development of APIs with Other
+  - Building static front end applications
+  - Deploying front end applications (Next.js, Remix, etc...)
+  - Script and automation (bots, scrapers)
+  - Proxy
+  - other
+
+## What Node.js binaries do you use
+
+- What is your Operating System for your local development environment ? (single choice - Other should give an input)
+  - Linux
+  - Windows
+  - macOS
+  - Linux with Docker
+  - Windows with Docker
+  - macOS with Docker
+  - Windows with WSL
+  - Other?
+
+- What is your Operating System in which you are running Node.js in production ? (multiple choice - Other should give an input)
+  - Linux/Unix
+  - Windows
+  - macOS
+  - Linux with Docker
+  - Windows with Docker
+  - macOS with Docker
+  - Other?
+
+- What architecture is the machine you are running Node.js in production? (multiple choice - Other should give an input)
+  - x64
+  - x32
+  - arm
+  - Don't know
+  - Other?
+
+- How do you get your `node` executables? (multiple choice – Other should give an input)
+  - With a Node.js version manager: nvm, n, nave, nvs, volta, etc.
+  - Using the official installer
+  - Using package managers like npm or yarn and install Node.js as a package
+  - Using a system package manager: apt-get, brew, dnf, yum …
+  - Downloading binaries directly from Nodejs.org
+  - Building Node.js from Source
+  - Other
+
+- What package manager do you use ? (multiple choice - Other should give an input)
+  - npm
+  - yarn v1
+  - yarn modern (2+)
+  - pnpm
+  - Other?
+
+- Which version manager do you use (multiple choice - Other should give an input)
+  - none
+  - nvm
+  - n
+  - asdf
+  - fnm
+  - nodenv
+  - volta
+  - Other?
+
+- How do you manage the package manager for your project? (multiple choice – Other should give an input)
+  - I use one version installed globally for all my projects.
+  - I use a tool to pick a specific version per project (e.g. Corepack, asdf, …).
+  - I use containers.
+  - Other?
+
+## Project Priorities and Direction
+
+- Do we want to keep the same question as last year?
```

Comment:
maybe some related to "have you tried Node.js's support for TypeScript" and maybe what are your remaining pain points in using ESM?